src/pages/courseInstanceAnalytics/courseInstanceAnalytics.js

```diff
@@ -0,0 +1,34 @@
+// const ERR = require('async-stacktrace')
+const router = require('express').Router({ mergeParams: true })
+const { sqlLoader } = require('@prairielearn/prairielib')
+const dbDriver = require('../../dbDriver')
+const asyncErrorHandler = require('../../asyncErrorHandler')
+// const checks = require('../../auth/checks')
+
+const sql = sqlLoader.loadSqlEquiv(__filename)
+
+router.get(
+  '/',
+  asyncErrorHandler(async (req, res, _next) => {
+    const { courseInstanceId } = req.params
```

Comment:
auth protect this route

Reply:
I assume we will be using middleware to protect all routes in the end @nwalters512 ?